libminifi/src/c2/ControllerSocketProtocol.cpp

```diff
@@ -325,33 +370,33 @@ void ControllerSocketProtocol::handleDescribe(io::BaseStream *stream) {
   }
 }
 
-void ControllerSocketProtocol::handleCommand(io::BaseStream *stream) {
+asio::awaitable<void> ControllerSocketProtocol::handleCommand(std::unique_ptr<io::BaseStream>&& stream) {
```

Comment:
what is the purpose of `$$`?

Reply:
It's unnecessary here, fixed in ad6de604481ec7e3b9349dcb89ab5c0a7263a39b